swap-p2p/src/protocols/swap_setup/bob.rs

```diff
@@ -437,11 +408,18 @@ impl ConnectionHandler for Handler {
     }
 
     fn on_behaviour_event(&mut self, new_swap: Self::FromBehaviour) {
+        tracing::trace!(
+            swap_id = %new_swap.swap_id,
+            "Received a new swap setup request from the Behaviour",
+        );
+
+        // TODO: Will the handler be woken up by this?
```

Comment:
redundant comment, connection handlers are always polled after an event from the behaviour